cmd/gtk/main.go

```diff
@@ -33,6 +34,10 @@ func init() {
 	passwordOpt = flag.String("password", "", "wallet password")
 	testnetOpt = flag.Bool("testnet", false, "initializing for the testnet")
 	version.NodeAgent.AppType = "gui"
+	if runtime.GOOS == "darwin" {
+		runtime.UnlockOSThread()
```

Comment:
Any reason for this change? Can you put some comment here

Reply:
@b00f done